### DISCUSSION TITLE
Where to get OpenJDK x86?

### DISCUSSION THREAD

**Discussion by Microsoft-Bob** [microsoft/openjdk#327] [Q&A] [answered]
According to the [support roadmap](https://learn.microsoft.com/en-us/java/openjdk/support), OpenJDK x86 is supported until January 2023, but I cannot find anywhere to download it.   Where is it?

**Comment by karianna** [comment]
Please drop us a note to ***@***.*** and we can likely
sort it out 🙂

On Fri, 21 Oct 2022 at 18:13, Microsoft-Bob ***@***.***>
wrote:

> According to the support roadmap
> <https://learn.microsoft.com/en-us/java/openjdk/support>, OpenJDK x86 is
> supported until January 2023, but I cannot find anywhere to download it.
> Where is it?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/microsoft/openjdk/discussions/327>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABME2ASSZZ6IWTJSV72YLLWELFKNANCNFSM6AAAAAARLLLYOA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Cheers, Martijn (Sent from Gmail Mobile)

**Reply by Microsoft-Bob** [reply]
Hi Martijn,

1) In your reply, your email address appears to have been obfuscated with ***@***.***

2) What's the reason to drop a separate note to you?  I'm dropping a note here to find out the answer to this seemingly simple question.

Cheers,
-Bob

**Reply by karianna** [reply]
@Microsoft-Bob was just a faster way of getting the full team on the case (we always update these support issues regardless).  We don't have a public download URI advertised for the x86-32 binaries but we can make one available to you.  If you drop us a note to openjdk-support@microsoftDOTcom then we'll get that to you.

**Comment by stdin82** [comment]
Just change x64 to x86 in the static link for Windows x64 zip/msi
https://learn.microsoft.com/en-us/java/openjdk/download

**Reply by brunoborges** [reply]
@stdin82 Not sure how you got a binary here, since we are not officially publishing builds for Windows 32 bit.

**Comment by karianna** [comment]
@Microsoft-Bob did that URI work for you?  I think in the latest release cycle we may have removed that access.

**Comment by brunoborges** [comment, marked as answer]
To clarify and close this discussion, we do not publish x86 binaries of Microsoft Build of OpenJDK for any operating system.
